domains/transfer/Shared/DataContext/Migrations/20240409063522_RemoveTransferAgreementHistory.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DataContext.Migrations
+{
+    /// <inheritdoc />
+    public partial class RemoveTransferAgreementHistory : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "TransferAgreementHistoryEntries");
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
```

Comment:
Don't think we should have a down function here. If we have dropped the table, the data is gone forever.